CliClient/tests/services_rest_Api.js

```diff
@@ -189,21 +189,62 @@ describe('services_rest_Api', function() {
 		done();
 	});
 
+	it('should create todos', async (done) => {
+		let response = null;
+		const f = await Folder.save({ title: "stuff to do" });
+
+		response = await api.route('POST', 'notes', null, JSON.stringify({
+			title: 'testing',
+			parent_id: f.id,
+			is_todo: 1
+		}));
+		expect(response.title).toBe('testing');
+		expect(!!response.id).toBe(true);
+		expect(response.is_todo).toBe(1);
+
+		response = await api.route('POST', 'notes', null, JSON.stringify({
+			title: 'testing 2',
+			parent_id: f.id,
+			is_todo: 0
+		}));
+		expect(response.title).toBe('testing 2');
+		expect(!!response.id).toBe(true);
+		expect(response.is_todo).toBeUndefined();
+
+		response = await api.route('POST', 'notes', null, JSON.stringify({
+			title: 'testing 3',
+			parent_id: f.id,
+		}));
+		expect(response.title).toBe('testing 3');
+		expect(!!response.id).toBe(true);
+		expect(response.is_todo).toBeUndefined();
+
+		response = await api.route('POST', 'notes', null, JSON.stringify({
+			title: 'testing 4',
+			parent_id: f.id,
+			is_todo: '1'
+		}));
+		expect(response.title).toBe('testing 4');
+		expect(!!response.id).toBe(true);
+		expect(response.is_todo).toBe(1);
```

Comment:
Thanks for adding the tests. To keep it short, please only check the value of "is_todo" (the value of title, id, etc. is already tested in other tests)